packages/react-dom/src/server/DOMMarkupOperations.js

```diff
@@ -30,12 +31,33 @@ import quoteAttributeValueForBrowser from './quoteAttributeValueForBrowser';
  * @param {*} value
  * @return {?string} Markup string, or null if the property was invalid.
  */
-export function createMarkupForProperty(name: string, value: mixed): string {
-  const propertyInfo = getPropertyInfo(name);
-  if (name !== 'style' && shouldIgnoreAttribute(name, propertyInfo, false)) {
+export function createMarkupForProperty(
```

Comment:
This function is inlined into `pushAttribute` in `ReactDOMServerFormatConfig`. So I'd expect them to have to change in sync. Is there some reason why that wasn't needed?

And if it _was_ needed, then do we know why none of the tests have failed despite missing logic there? This seems to point to missing coverage.

Reply:
So there's two things here.

- It appears this codepath doesn't run anymore at all. This is likely because `ReactPartialRenderer` isn't used in open source at all anymore. It's only used at FB while we haven't switched on new SSR engine yet. So we should probably be careful here before merge. I'll check if we can disable the old renderer so that we're not scared to merge without test coverage.

- The codepath in `ReactDOMServerFormatConfig` _does_ run — but it doesn't have these changes (so it's probably wrong). And because we didn't have a test written specifically to verify these changes, we missed this. So we need to add a test that verifies all of the SSR behavior we care about. You can either use our SSR integration suite (and add more cases there, potentially copy-pasting the Attributes file into a separate one focused on Custom Elements) or you can add more "manual" cases to `ReactServerRendering-test.js` or nearby. But we should make sure to have a regression test for all the design decisions we've made about the output.

Reply:
Thanks for catching this!
I removed the changes to DOMMarkupOperations and implemented the proposed SSR changes in ReactDOMServerFormatConfig and added tests in ReactServerRendering-test.